pyproject.toml

```diff
@@ -1,7 +1,55 @@
+[project]
+name = "weasel"
+version = "0.0.0"
+authors = [
+    {name = "Explosion", email = "contact@explosion.ai"},
+]
+description = "Weasel: A small and easy workflow system"
+readme = "README.md"
+requires-python = ">=3.8"
```

Comment:
Unrelated to this PR, but isn't this a problem since we support Python 3.6+ in spaCy?

Reply:
I might be mistaken, but I think spaCy v4 will drop support for Python 3.6 and 3.7 since they've reached end-of-life (Python 3.7 will reach EOL on the 27th).

It makes things much easier for us, we no longer have to rely on brittle backports.

Reply:
Ah, ok. In that case this should target the v4 branch.

Reply:
I don't think we want to make this change as part of this PR right now - for now we should just keep supporting 3.6 and 3.7 - the backports are all already there. Once we do decide to drop support from spaCy, it won't be difficult to just sweep across and clean up.